scripts/build-operator

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+source $(dirname $0)/lib/debug_functions
+
+cd $(dirname $0)
+
+GOFLAGS=""
```

Comment:
This necessary? If so, shouldn't it be exported?

Reply:
hmmm good question, since it's not exported I suspect it's not?